plugins/Sid/SidInstrument.cpp

```diff
@@ -706,15 +705,15 @@ void SidInstrumentView::updateKnobToolTip()
 	SidInstrument * k = castModel<SidInstrument>();
 	for( int i = 0; i < 3; ++i )
 	{
-		ToolTip::add( m_voiceKnobs[i].m_sustKnob,
+		 m_voiceKnobs[i].m_sustKnob->setToolTip(
 				QString::number( (int)k->m_voice[i]->m_sustainModel.value() ) );
-		ToolTip::add( m_voiceKnobs[i].m_crsKnob,
+		 m_voiceKnobs[i].m_crsKnob->setToolTip(
 				QString::number( (int)k->m_voice[i]->m_coarseModel.value() ) +
 				" semitones" );
 	}
-	ToolTip::add( m_volKnob,
+	 m_volKnob->setToolTip(
 					QString::number( (int)k->m_volumeModel.value() ) );
-	ToolTip::add( m_resKnob,
+	 m_resKnob->setToolTip(
```

Comment:
The space after the opening parenthesis wasn't deleted for each of these lines, so they are indented an extra space.